.github/workflows/devcontainer-image-build.yaml

```diff
@@ -2,6 +2,11 @@ name: Bucketeer Dev Container Build and Push Image
 
 on:
   workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - ".github/.devcontainer"
```

Comment:
Any changes in this directory will trigger the workflow when merging to the main branch.

Reply:
If the other files in this directory don't affect the docker image when they change, we can restrict the trigger to the `Dockerfile` file only.

Reply:
The other files in this directory is the key point to change, `.github/.devcontainer` is fine.

Generally we need to add tools in `devcontainer.json` then build the image.